rotkehlchen/tests/unit/test_graph.py

```diff
@@ -0,0 +1,146 @@
+from contextlib import ExitStack
+from http import HTTPStatus
+from unittest.mock import MagicMock, patch
+
+import pytest
+from requests import HTTPError, Response
+
+from rotkehlchen.chain.ethereum.graph import RETRY_STATUS_CODES, Graph, format_query_indentation
+from rotkehlchen.constants.timing import QUERY_RETRY_TIMES
+from rotkehlchen.errors import RemoteError
+
+TEST_URL_1 = 'https://api.thegraph.com/subgraphs/name/uniswap/uniswap-v2'
```

Comment:
These unit tests don't really make any network queries, right?

Reply:
Not now as we deactivated `fetch_schema_from_transport`, otherwise there would be a request when `Graph` is instantiated.

Reply:
Okay thanks.